added namespace enumerated values accessor

bmcfee opened the pull request:
This PR adds an accessor function for enumerated namespaces (eg, `tag_gtzan`).

Example usage:

```python
>>> values = jams.schema.values('tag_gtzan')
['blues', 'classical', 'country', ...]
```

This is useful for supervised learning tasks, where it's helpful to know the vocabulary up front (eg, for building label encoders).

It only works/makes sense if the namespace's `values` field is an `enum` type.  This rules out things like `tag_open` or `chord`, which are defined by regular expressions (and are, in effect, infinite vocabularies).  If the given namespace is not enumerated, a `NamespaceError` is thrown.  Otherwise, a copy of the values array is returned.

This should be ready for CR.

bmcfee commented:
@ejhumphrey CR plz?

bmcfee commented:
![crickets](http://i.giphy.com/l2R013mIf1ZXdvoyI.gif)

rabitt commented on jams/schema.py:
```diff
@@ -88,6 +88,33 @@ def is_dense(ns_key):
 
 
+def values(ns_key):
+    '''Return the allowed values for an enumerated namespace.
```
An example here would be helpful.

rabitt commented:
👍 Everything looks good. Only thing I changed was I added the gtzan example to the values function docstring.

bmcfee commented:
> Only thing I changed was I added the gtzan example to the values function docstring.

thanks! I updated the doc build to include this and fixed a typo; otherwise this is mergeable.
